aggregator/src/aggregator.rs

```diff
@@ -2098,33 +2098,33 @@ impl VdafOps {
                     }
 
                     // Write report shares, and ensure this isn't a repeated report aggregation.
+                    // TODO(#225): on repeated aggregation, verify input share matches previously-received input share
                     try_join_all(report_share_data.iter_mut().map(|rsd| {
                         let task = Arc::clone(&task);
                         let aggregation_job = Arc::clone(&aggregation_job);
 
                         async move {
                             // Verify that we haven't seen this report ID and aggregation parameter
                             // before in another aggregation job.
```

Comment:
nit: this comment doesn't reflect how `put_report_share_fut` works, it belongs next to `report_aggregation_exists_fut`.